script.js

```diff
@@ -0,0 +1,197 @@
+let todoList = [];
+let doneList = [];
+
+// 입력된 값을 todo list에 추가
+const addTodoItem = () => {
+  const todoContent = document.querySelector(".todo-input").value;
+  if (todoContent) paintTodoItem(todoContent);
+};
+
+// 화면에 todo item을 추가
+const paintTodoItem = (text) => {
+  const todoId = new Date().valueOf();
+  const newTodo = document.createElement("li");
+  const newTodoText = document.createElement("span");
+  const todoDel = document.createElement("img");
+
+  // todo item 텍스트 추가
+  newTodoText.innerText = text;
+  newTodoText.style.cursor = "pointer";
+  newTodoText.style.fontSize = "15px";
+  newTodoText.style.paddingLeft = "5px";
+  newTodoText.addEventListener("dblclick", toggleTodoToDone);
+
+  // todo item 삭제 버튼 추가
+  todoDel.setAttribute("src", "img/bin.png");
+  todoDel.style.width = "14px";
+  todoDel.style.paddingLeft = "5px";
+  todoDel.style.cursor = "pointer";
+  todoDel.addEventListener("click", deleteTodoItem);
+
+  // li에 item 추가
+  newTodo.setAttribute("id", todoId);
+  newTodo.style.marginBottom = "13px";
+  newTodo.appendChild(newTodoText);
+  newTodo.appendChild(todoDel);
+
+  // todo list에 item 추가
+  const todoObj = {
+    id: todoId,
+    text,
+  };
+  todoList.push(todoObj);
+  document.querySelector(".todo-list").appendChild(newTodo);
+  document.querySelector(".todo-input").value = "";
+
+  // todo item 개수 갱신
+  countTodoItem();
+
+  // localStorage에 갱신된 todoList 저장
+  saveTodoLocalStorage();
+};
+
+// todo item을 삭제
+const deleteTodoItem = (e) => {
+  const target = e.target.parentNode;
+  document.querySelector(".todo-list").removeChild(target);
+
+  // target node 삭제
+  const newTodoList = todoList.filter(
+    (item) => item.id !== parseInt(target.id),
+  );
+  todoList = newTodoList;
+
+  // todo item 개수 갱신
+  countTodoItem();
+
+  // localStorage에 갱신된 todoList 저장
+  saveTodoLocalStorage();
+};
+
+// todo item 개수 출력
+const countTodoItem = () => {
+  const count = document.querySelector(".todo-count");
+  count.innerText = "(" + todoList.length + ")";
```

Comment:
템플릿 리터럴을 이용해서 표현할 수 있다고 합니다! 
```suggestion
  count.innerText = `(${todoList.length}`;
```

Reply:
아하 알려주셔서 감사합니다~!